packages/spark-core/base/_links.scss

```diff
@@ -14,21 +15,35 @@
   &:hover,
   &:active,
   &:focus {
+    color: $red-real;
     text-decoration: none;
   }
+
+  &::after {
+    content: '';
+    top: 100%;
+    left: 0;
+    height: 2px;
```

Comment:
New: This is what it looks like with this new change on a larger font size. 
![screen shot 2018-10-19 at 8 32 06 am](https://user-images.githubusercontent.com/2117593/47218530-0c432d00-d37a-11e8-9127-b7c54e2681b2.png)

Old: This is what it looks like with the old text-decoration way.
![screen shot 2018-10-19 at 8 34 46 am](https://user-images.githubusercontent.com/2117593/47218663-8a073880-d37a-11e8-807e-882b164066e3.png)

The underline used to scale. Do we want the new line to scale with the text (i.e. do we want to set it in ems)?